package.json

```diff
@@ -93,11 +93,14 @@
       "json",
       "ts"
     ],
-    "rootDir": "src",
+    "rootDir": "libs",
```

Comment:
Not a good idea, it would ignore the tests in `src`. Let's keep it like that as we're moving the test to `src` anyway